### ISSUE TITLE
Notifications channel order

### ISSUE THREAD

**Issue by manton** [indieweb/microsub#46]
A couple places in the spec it says the notifications channel must be first in the list:

> When returning the list of channels, the notifications channel must always be the first in the list, as clients are expected to treat it separately such as showing it as a separate icon, not in the main channel list.

Is this really a necessary requirement? I never noticed this and I've been returning the notifications channel as the 2nd item. It seems to work fine with Microsub clients I've tested. Because the channel has a special `uid`, a client can take it out of the list and treat it separately if it wants, or not, and the order shouldn't matter.

Is there another reason it needs to be first, or can we update the spec to drop that requirement?

**Comment by jalcine** [comment]
Yeah, I think there isn't an explicit need. The UIDs are enough if sorting is needed.
